homeassistant/components/nederlandse_spoorwegen/strings.json

```
@@ -0,0 +1,144 @@
+          "name": "A name for this route.",
+          "from": "Select the departure station",
+          "to": "Select the destination station",
+          "via": "Optional intermediate station",
+          "time": "Optional departure time in HH:MM or HH:MM:SS (24h)"
+        }
+      },
+      "remove_route": {
+        "title": "Remove route",
+        "description": "Select a route to remove:",
+        "data": {
+          "route_to_remove": "Route to remove"
+        }
+      }
+    },
+    "error": {
+      "cannot_connect": "[%key:component::nederlandse_spoorwegen::config::error::cannot_connect%]",
+      "invalid_auth": "[%key:common::config_flow::error::invalid_api_key%]",
+      "missing_fields": "[%key:component::nederlandse_spoorwegen::config::error::missing_fields%]",
+      "no_stations_available": "[%key:component::nederlandse_spoorwegen::config::error::no_stations_available%]",
+      "same_station": "[%key:component::nederlandse_spoorwegen::config::error::same_station%]",
+      "invalid_station": "[%key:component::nederlandse_spoorwegen::config::error::invalid_station%]",
+      "invalid_time_format": "[%key:component::nederlandse_spoorwegen::config::error::invalid_time_format%]",
+      "unknown": "[%key:common::config_flow::error::unknown%]"
+    }
+  },
+  "entity": {
+    "sensor": {
+      "departure_platform_planned": {
+        "name": "Departure platform planned"
+      },
+      "departure_platform_actual": {
+        "name": "Departure platform actual"
+      },
+      "arrival_platform_planned": {
+        "name": "Arrival platform planned"
+      },
+      "arrival_platform_actual": {
+        "name": "Arrival platform actual"
+      },
+      "departure_time_planned": {
+        "name": "Departure time planned"
+      },
+      "departure_time_actual": {
+        "name": "Departure time actual"
+      },
+      "arrival_time_planned": {
+        "name": "Arrival time planned"
+      },
+      "arrival_time_actual": {
+        "name": "Arrival time actual"
+      },
+      "next_departure": {
+        "name": "Next departure"
+      },
+      "status": {
+        "name": "Status"
+      },
+      "transfers": {
+        "name": "Transfers"
+      },
+      "route_from": {
+        "name": "Route from"
+      },
+      "route_to": {
+        "name": "Route to"
+      },
+      "route_via": {
+        "name": "Route via"
+      }
+    }
+  },
+  "issues": {
+    "integration_yaml_migration": {
+      "title": "Nederlandse Spoorwegen YAML configuration migrated",
+      "description": "Your Nederlandse Spoorwegen integration configuration has been automatically migrated from YAML to the new config entry format. To complete this migration: 1) Verify the integration is working correctly in the integrations page, 2) Remove the 'nederlandse_spoorwegen:' section from your configuration.yaml file, 3) Restart Home Assistant to clear this notification. The integration will continue to work normally during this transition."
```

We should be able to use markup (or at least some line breaks if markup does not work in the Repairs UI) for the numbered lists for nicer formatting:

```suggestion
      "description": "Your Nederlandse Spoorwegen integration configuration has been automatically migrated from YAML to the new config entry format.\nTo complete this migration:\n1. Verify the integration is working correctly in the integrations page.\n2. Remove the 'nederlandse_spoorwegen:' section from your configuration.yaml file.\n3. Restart Home Assistant to clear this notification.\nThe integration will continue to work normally during this transition."
```

At least this resolves the wrong commas at the end of the numbered lines.